### ISSUE TITLE
Real World Testing project missing tag on list view

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#1208]
On the project list we should show the project type as "Real World Testing". Currently the type tag is missing.

![screen shot 3](https://user-images.githubusercontent.com/8484061/31004112-c5e1d5ca-a4a7-11e7-96a3-b2ed36aee738.png)

**Comment by vikasrohit** [comment]
I have already raised this few days back. #1197 and you closed it saying it is a minor issue. 😬 

Now, assuming that we have to show actual project type when rendering the tags in the list view. In past it used to be a mapping from project type to the text which we want to show because we used different text than project's type here it is: 

<pre>
'generic': 'Work Project',
  'visual_design': 'Design',
  'visual_prototype': 'Design & Prototype',
  'app_dev': 'Full App'
</pre>

Now to get it quick fixed, I am assuming we are going to use project type as text for the label and green color for all new project types which are App, Website, Chatbot and 'Real World Testing'.
